app/api/public-cloud/aws-roles/getGroupRolesNames/route.ts

```diff
@@ -0,0 +1,27 @@
+import { getGroupsNamesByLicencePlate } from '@/app/api/public-cloud/aws-roles/helpers';
+import { NextResponse } from 'next/server';
+import { z } from 'zod';
+import createApiHandler from '@/core/api-handler';
+
+interface QueryParam {
+  licencePlate: string;
+}
+
+const queryParamSchema = z.object({
+  licencePlate: z.string(),
+});
+
+const apiHandler = createApiHandler<unknown, QueryParam>({
+  roles: ['user'],
+  validations: { queryParams: queryParamSchema },
+});
+
+export const GET = apiHandler(async ({ queryParams }) => {
+  const { licencePlate } = queryParams;
+  let result;
+  if (licencePlate) {
```

Comment:
since the query param validation schema has `licencePlate`. I don't think there is a case that `licencePlate` is falsy.

Reply:
removed validation for `licencePlate`